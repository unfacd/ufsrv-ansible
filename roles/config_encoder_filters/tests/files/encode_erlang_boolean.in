- var1: yes
- var2: false
- var3: "True"
- var4: "false"
