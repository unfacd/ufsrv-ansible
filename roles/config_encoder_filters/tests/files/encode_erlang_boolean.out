[
  {var1, true},
  {var2, false},
  {var3, "True"},
  {var4, "false"}
].
